oggm_3dviz/tools/viz.py

```diff
@@ -550,29 +550,78 @@ def update_glacier(self, step, camera_position_per_step=None):
         self.plotter.update()
 
     def export_animation(self, filename="animation.mp4", framerate=10,
-                         quality=5, moving_camera_start_and_end_point=None,
+                         quality=5, camera_trajectory='linear', linear_camera_start_and_end_point=None,
+                         rotate_camera_start_and_end_angle=None, rotate_camera_height=5, rotate_camera_radius=1,
                          **kwargs):
+        """
+        Export an animation of the glacier model.
```

Comment:
I really appreciate that you have added a docstring to this function. Could you adapt the format of the Parameters to use the same style as in the other docstrings in the project. E.g.  :
```
filename: str
    The name of the output video file. Defaults to "animation.mp4".
```